### PR TITLE
Add content 3

### PR DESCRIPTION
**Pull request by Aichnerc**
**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [ ] Have you updated or added documentation for the change?
 - [x] Have you tested your changes with successful results?

**Type of Changes:**

 
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [x] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change)

**What is the current behavior? (link to any open issues here)**
- Currently there is no way to dynamically generated data. (Basic CMS support) #32 
- Currently there is no "Feature" section. #33 

**What is the new behavior (if this is a feature change)?**
- A way to add dynamic generated data has been added. #32 
- A "Feature" section has been added. #33

### CODE REVIEW

**Review by Aichnerc — COMMENTED** [review]
Looks okay to me.
The long picture URL is temporary and will be replaced with the next update.

File: src/components/organisms/Sections/About/index.jsx [hunk 1]
```diff
@@ -20,69 +20,92 @@ import './about.scss';
...
+        );
+      } else {
+        return undefined;
+      }
+    });
+    return rtn;
+  }
+
+  render(){
+
```
Comment (kleberbaum) [inline src/components/organisms/Sections/About/index.jsx]: whitespace?

**Review by kleberbaum — CHANGES_REQUESTED** [review]
should be ok to merge after a few changes.

**Review by kleberbaum — APPROVED** [review]
should be ok to merge.

**Pull request merged**